client-config/src/main/java/com/palantir/conjure/java/client/config/ClientConfigurations.java

```diff
@@ -48,6 +48,8 @@
     private static final int DEFAULT_MAX_NUM_RETRIES = 4;
     private static final ClientConfiguration.ClientQoS DEFAULT_DISABLE_SYMPATHETIC_QOS =
             ClientConfiguration.ClientQoS.ENABLED;
+    private static final ClientConfiguration.PropagateQoS DEFAULT_ENABLE_PROPAGATE_QOS =
+            ClientConfiguration.PropagateQoS.DISABLED;
```

Comment:
Should this be named `DEFAULT_DISABLE_PROPAGATE_QOS`?

Reply:
I'm not entirely sure. I based this name off of the 2 lines above:
```
    private static final ClientConfiguration.ClientQoS DEFAULT_DISABLE_SYMPATHETIC_QOS =
            ClientConfiguration.ClientQoS.ENABLED;
```
Note that the default value is `ENABLED` and the name of the constant is `DEFAULT_DISABLE_...`. Since propagate QoS is `DISABLED` by default, I named the constant `DEFAULT_ENABLE_...`

Reply:
+1 to @schlosna's suggestion - I think the variable name on line 49 doesn't make sense to me - renaming to something like CLIENT_QOS_DEFAULT and PROPAGATE_QOS_DEFAULT seem ok?

Reply:
OK that sounds good. I'll rename the default for `PropogateQoS` in this PR and rename the default for `ClientQoS` in a separaten one.

Reply:
https://github.com/palantir/conjure-java-runtime/pull/1054